dashboard/app/models/contact_rollups_raw.rb

```diff
@@ -33,6 +88,36 @@ def self.extract_parent_emails
     ContactRollupsV2.execute_query_in_transaction(query)
   end
 
+  def self.extract_scripts_taught
+    source_sql = <<~SQL
+      SELECT email, scripts.name, MAX(se.updated_at) AS updated_at
+      FROM users u
+      JOIN sections se ON se.user_id = u.id
+      JOIN scripts ON scripts.id = se.script_id
+      WHERE scripts.name in (#{CSF_SCRIPT_LIST})
+      AND email > ''
+      GROUP BY email, scripts.name
+    SQL
+
+    query = get_extraction_query(source_sql, 'email', ['name'], true, 'dashboard.sections.script_id')
```

Comment:
How about extracting `script_name` from `dashboard.sections` table? It will be cleaner to write its processing code later.

The courses_taught below will be `course_name` from `dashboard.sections`.

Reply:
hmm I don't understand, `script_name` isn't a field in `sections`?

Reply:
What I'm really thinking about is doing away with the `db.table.column` naming convention, and just use `db.table`. (Since we allow multiple `email`-`sources` in the contact_rollups_raw table now.) The latter is shorter and makes more sense when refers to as a data source. 

Later in the processing step, we can say: extract `script_name` or `course_name` from `dashboard.sections`. Granted, script_name and course_name are not real columns in the sections table, but are derived from script_id and course_id.